src/java.base/share/classes/java/util/Random.java

```diff
@@ -80,8 +81,143 @@
         name = "Random",
         i = 48, j = 0, k = 0,
         equidistribution = 0
-)
+        )
 public class Random implements RandomGenerator, java.io.Serializable {
+
+    /**
+     * Class used to wrap a {@link java.util.random.RandomGenerator} to
+     * {@link java.util.Random}.
+     */
+
+    @SuppressWarnings("serial")
```

Comment:
Is this really fine, or should a `serialVersionUID` be defined?
(And maybe `initialized` should be made transient then)